webrob/pages/main.py

```diff
@@ -127,10 +127,10 @@ def render_change_password_post():
     #return render_template('main.html', **locals())
 
 # method to send woff font files from node_modules
-@app.route('/user/<node_modules>/<typeface_oswald>/<files>/<path:file_name>')
-@app.route('/<node_modules>/<typeface_oswald>/<files>/<path:file_name>')
-def send_woff_file_from_neem_directory(node_modules, typeface_oswald, files, file_name):
+@app.route('/user/node_modules/<typeface_oswald>/<files>/<path:file_name>')
```

Comment:
I wonder, isn't it enough to use the following routes:

@app.route('/user/node_modules/<path:file_path>')
@app.route('/node_modules/<path:file_path>')

This would basically allow to download any file from the node_modules, no matter where it is.

Reply:
Got it. Please check latest commit message.